app/views/notifications/index.html.erb

```diff
@@ -15,7 +15,7 @@
           <% if @notifications.to_a.any? %>
             <div class="per-page btn-group float-right d-none d-md-block" data-toggle="tooltip" data-placement="left" title="Notifications per page">
               <button type="button" class="btn btn-sm btn-outline-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                <%= per_page = params[:per_page].presence || 20 %>
+                <%= per_page = @per_page || 20 %>
```

Comment:
It looks to me like we can just delete this line altogether and replace `per_page.to_i` on line 23 with `@per_page`